### DISCUSSION TITLE
Auto export to filesystem?

### DISCUSSION THREAD

**Discussion by ippocratis** [TrackMyIndoorWorkout/TrackMyIndoorWorkout#520] [Q&A]
I have enabled the preferences>user experience>instant export

Also specified the export Dir in preferences>user experience>paste an export folder

In my case 

/storage/emulated/0/Workout

Nothing is auto exported 

I have to manually do it

Do I miss something?

**Comment by MrCsabaToth** [comment]
I suspect that there's something with that path, the app probably cannot access it. At some point I had a logic which could deduct the folder from a successful export, but things went through several revisions, so I need to check.

**Reply by MrCsabaToth** [reply]
Ref https://github.com/TrackMyIndoorWorkout/TrackMyIndoorWorkout/issues/438#issuecomment-1631908187
